modules/RTC/TraceablePeerConnection.js

```diff
@@ -328,12 +326,26 @@ export default function TraceablePeerConnection(
     this._localTrackTransceiverMids = new Map();
 
     /**
-     * Holds the SSRC map for the local tracks.
+     * Holds the SSRC map for the local tracks mapped by their source names.
      *
-     * @type {Map<string, TPCSSRCInfo>}
+     * @type {Map<string, TPCSourceInfo>}
+     * @property {string} msid - The track's MSID.
+     * @property {Array<string>} ssrcs - The SSRCs associated with the track.
+     * @property {Array<TPCGroupInfo>} groups - The SSRC groups associated with the track.
      */
     this._localSsrcMap = null;
```

Comment:
Any reason not to initialize this to a new map too?

Reply:
I have some more work planned around this, i.e., getting rid of localSSRCs and remoteSSRCs in favor of these. Lets restrict this PR to only the SDP conversion, WDYT?

Reply:
It just looks odd that one is a map and the other one is null. If it's not a huge change it would be nice to at least have them aligned. No worries if it would break things, we can do it later.